squidDrone/src/types/propulsion_hardware_types.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef SRC_TYPES_PROPULSION_HARDWARE_TYPES_HPP_
+#define SRC_TYPES_PROPULSION_HARDWARE_TYPES_HPP_
+
+
```

Comment:
One empty line to much. But no real problem here.

Reply:
Fixed in [this commit](https://github.com/elheck/Squiddrone/pull/230/commits/e04970a97abe893523fc4550125cd271e221a661).